translations/monero-core_ar.ts

```diff
@@ -56,46 +56,46 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="103"/>
         <source>Can&apos;t create entry</source>
-        <translation>Can&apos;t ينشا مدخلات</translation>
+        <translation>&apos;لا يمكن  إنشاء البيانات</translation>
     </message>
 </context>
 <context>
     <name>AddressBookTable</name>
     <message>
         <location filename="../components/AddressBookTable.qml" line="47"/>
         <source>No more results</source>
-        <translation>لا نتائج جديده</translation>
+        <translation>لايوجد نتائج اخري</translation>
     </message>
     <message>
         <location filename="../components/AddressBookTable.qml" line="105"/>
         <source>Payment ID:</source>
-        <translation>هويه المعامله</translation>
+        <translation>هويه المعامله:</translation>
     </message>
     <message>
         <location filename="../components/AddressBookTable.qml" line="150"/>
         <source>Address copied to clipboard</source>
-        <translation type="unfinished"></translation>
+        <translation>تم نسخ العنوان إلى الحافظة</translation>
     </message>
 </context>
 <context>
     <name>DaemonConsole</name>
     <message>
         <location filename="../components/DaemonConsole.qml" line="177"/>
         <source>command + enter (e.g help)</source>
-        <translation type="unfinished"></translation>
+        <translation>الأمر + زر انتر (مثال help)</translation>
     </message>
 </context>
 <context>
     <name>DaemonManagerDialog</name>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="93"/>
         <source>Starting local node in %1 seconds</source>
-        <translation type="unfinished"></translation>
+        <translation>بدأ تشغيل الخادم المحلي في خلال %1 ثواني</translation>
     </message>
     <message>
         <location filename="../components/DaemonManagerDialog.qml" line="111"/>
         <source>Start daemon (%1)</source>
-        <translation>إبدأ الخادم</translation>
+        <translation> إبدأ الخادم (1%)</translation>
```

Comment:
The place marker should be inverted: `(1%)`

Reply:
it's alright it looks like that cuz of the arabic words

Reply:
I edited everything else